.vscode/launch.json

```diff
@@ -32,8 +32,6 @@
 			"name": "Mocha All",
 			"program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
 			"args": ["--config", "test/.mocharc.yml", "--timeout", "0", "${workspaceFolder}/test/*.ts"],
-			"console": "integratedTerminal",
```

Comment:
Just double checking here, these are being removed because mocha display outputs fine, it's just when debugging Emcee that we need them, right?

Reply:
Mocha can render to the debug console just fine for me. If using the integrated terminal, on each run, a new one is opened, and soon we have a dozen sessions open from the programming cycle.